pymc3/tuning/starting.py

```diff
@@ -30,21 +34,21 @@ def find_MAP(start=None, vars=None, fmin=None,
     start : `dict` of parameter values (Defaults to `model.test_point`)
     vars : list
         List of variables to set to MAP point (Defaults to all continuous).
-    fmin : function
-        Optimization algorithm (Defaults to `scipy.optimize.fmin_bfgs` unless
+    method : string or callable
```

Comment:
I'm not sure.  Some benefits I can think of is it matches up a bit better with `sample` and `fit`, which specify the method with strings.  Also, `minimize` can take a homebrew optimizer, which may be nice.  It also gives a consistent interface for all scipys optimizers.  The different fmin_*` functions are sometimes a little bit different.  Draw back is breaking peoples code...